ConsoleGameLib/CoreTypes/Point.cs

```diff
@@ -64,5 +64,15 @@ public override bool Equals(object obj)
         {
             return new Point(-unary.X, -unary.Y);
         }
+
+        public static Point operator *(Point point, int scalar)
+        {
+            return new Point(point.X * scalar, point.Y * scalar);
+        }
+
+        public static Point operator /(Point point, int scalar)
+        {
+            return new Point(point.X / scalar, point.Y / scalar);
+        }
     }
```

Comment:
Adding/subtracting points is not really meaningful mathematically, but I understand we're treating these as Vectors essentially. If we later find a need for a Vector class, these methods should instead be implemented there. At the moment, it's convenient to have them though.